pkg/component/nodeexporter/monitoring.go

```diff
@@ -191,6 +191,9 @@ kubernetes_sd_configs:
   authorization:
     type: Bearer
     credentials_file: /var/run/secrets/gardener.cloud/shoot/token/token
+  namespaces:
+    names: [ kube-system ]
+  follow_redirects: false
```

Comment:
This line is not reflected in the unit test.

```
follow_redirects: false
```

A minor typo in the commit message: e1b462133e12b8e7429db775f69d9d33f7e638a7

> potential targets Prometheus has _to_ discard during relabeling.

Reply:
The `follow_redirects: false` needs to go the the scrape_config section.

Reply:
Thanks, done (same for the other ones).